go-controller/pkg/factory/handler.go

```diff
@@ -242,7 +276,7 @@ func (i *informer) getNewQueueNum(numEventQueues uint32) uint32 {
 	return queueIdx
 }
 
-func (i *informer) refQueueEntry(oType reflect.Type, obj interface{}, numEventQueues uint32) (ktypes.NamespacedName, *queueMapEntry) {
+func (qm *queueMap) refEntry(oType reflect.Type, obj interface{}) (ktypes.NamespacedName, *queueMapEntry) {
```

Comment:
can we change the names of refEntry and unrefEntry? I don't know where those names came from (hope it wasnt me haha), but I don't understand what they mean. Maybe it ref comes from refCount. Anyway could we change it to getQueueMapEntry or something?

Reply:
Changed to getQueueMapEntry / releaseQueueMapEntry